dashboard/src/pages/packages/index.vue

```diff
@@ -52,6 +66,26 @@ const toggleLegend = () => {
       {{ packageStore.page.total }}
     </div>
 
+    <div class="row mx-0 mb-3 align-items-left border bg-light">
+      <div class="col-auto pb-1 align-self-center border-end">
+        <h3 class="mb-0">Filters</h3>
+      </div>
+      <div class="col-auto pb-1">
+        <label for="filter-status" class="form-label mb-0"> Status </label>
```

Comment:
```html
<div class="d-flex flex-wrap gap-3 mb-3 p-3 border bg-light">
      <h3 class="mb-0 pe-3 border-end">Filters</h3>
      <div class="d-flex gap-2">
        <label for="filter-status" class="align-self-center">Status</label>
```

- On the first div:
  - Used `d-flex` instead of `row mx-0` and `col-` classes in the children.
  - Added `flex-wrap` for responsiveness.
  - Added `gap-3` for separation between elements (also works spacing vertically when the wrap kicks in).
  - Removed `align-items-left`, it should be `align-items-start` and it's the default.
  - Added padding `p-3`.
- Removed the div wrapping the heading, it's not needed.
- On the heading, just moved the border there and added padding to the end.
- On the div wrapping the label and select:
  - Added `d-flex gap-2`, same as the first div, smaller gap and not wrapping.
- On the label:
  - Removed `form-label mb-0`, the first one only adds margin bottom and the second one removes it.
  - Added `align-self-center` to center it vertically.


![image](https://github.com/user-attachments/assets/2ca79ef8-c814-4ae1-958a-dea7332fed7e)
![image](https://github.com/user-attachments/assets/8305851b-a3fc-4c03-941c-e7e50f699de3)